draft-ietf-snac-simple.xml

```diff
@@ -354,9 +354,14 @@
 	    <t>
 	      In this state, the stub router generates its own on-link prefix for the interface. This prefix has a valid and
 	      preferred lifetime of STUB_PROVIDED_PREFIX_LIFETIME seconds. The stub router sends a router advertisement containing
-	      this prefix. The 'A' (autonomous configuration), 'L' (on-link) <xref target="RFC4861" section="4.6.2"/> and the Stub
-              Router bit (<xref target="I-D.hui-stub-router-ra-flag"/>) MUST be set in the Router Advertisement header flags
-              field <xref target="RFC5175"/>.</t>
+	      this prefix in a Prefix Information Option (PIO). In the PIO, the 'A' (autonomous configuration) and 'L' (on-link) 
+	      flags <xref target="RFC4861" section="4.6.2"/> MUST be set. The Stub
+	      Router bit (<xref target="I-D.hui-stub-router-ra-flag"/>) MUST be set in the Router Advertisement header flags
+	      field <xref target="RFC5175"/>. In the Router Advertisement header, the values of the 'M' and 'O' flags MUST be copied
+	      from the respective M/O values seen in a recent Router Advertisement received from a non-stub-router, if any.
```

Comment:
"seen in the most recent"

Reply:
Maybe we should add "After a reboot, if no recent advertisements have been received from non-stub routers, but a recent advertisement has been received from a stub router, the values for the 'M' and 'O' bits provided by that stub router should be copied. After STALE_RA_TIME after reboot, the stub router should go back to the behavior of only using 'M' and 'O' bit values from non-stub routers. This avoids a situation where a stub router that has rebooted starts to advertise different 'M' and 'O' bits than other stub routers present on the link."

Reply:
Text for "after reboot" was included and reworded.